tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_prplwrt.py

```diff
@@ -6,19 +6,27 @@
 ###############################################################
 
 import boardfarm
-from boardfarm.devices import OpenWrtRouter
-from environment import ALEntityPrplWrt, _get_bridge_interface
+import json
+import os
+import pexpect
+import subprocess
+import sys
+
 from .prplmesh_base import PrplMeshBase
+from boardfarm.devices import openwrt_router, connection_decider
+from environment import ALEntityPrplWrt, _get_bridge_interface
+from ipaddress import IPv4Network
 from sniffer import Sniffer
 
 
-class PrplMeshPrplWRT(OpenWrtRouter, PrplMeshBase):
+class PrplMeshPrplWRT(openwrt_router.OpenWrtRouter, PrplMeshBase):
```

Comment:
Wrong "path" was used. To avoid the change, i have to change import in next way:
````
from boardfarm.devices.openwrt_router import OpenWrtRouter¬
````

Reply:
Fixed.